requirements.txt

```diff
@@ -0,0 +1,10 @@
+Pygments==2.2.0 
```

Comment:
I guess this is synthesised from the [log](https://readthedocs.org/api/v2/build/6171938.txt), but I think we can do without it.

I don't think there's anything fundamental about the versions there, nor do we seem to need tham all. I triggered a rebuild (changed the project name) recently and the versions in the [new log](https://readthedocs.org/api/v2/build/9015818.txt) are not the same. If we discover we need to pin dependencies then ok, but it has been enough for me to install the latest `sphinx` and `sphinx-rtd-theme`.

Reply:
I've cut it down to sphinx and sphinx-rtd-theme and removed pinned versions in the latest.  I guess if you feel strongly we can remove the whole `requirements.txt`, I usually like the environment dependencies to be pretty explicit.

Reply:
I'm happy to go with that thanks. Really, I don't know what is best here so am unable to "feel strongly". I see two possible strong positions:
1. Pin all the dependencies so we get exactly reproduceable results. It looks the same to a contributor as on RTD and if it needs fixing, he/she re-works `requirements.txt`.
1. Let RTD decide so we get the bug fixes and best platform compatibility. It won't run for a contributor until the dependencies are satisfied, and then with his/her different versions.

PyPA ought to know, and their documentation uses [a short `requirements.txt`](https://github.com/pypa/packaging.python.org/blob/master/requirements.txt) with version numbers.

If I've [understood correctly](https://caremad.io/posts/2013/07/setup-vs-requirement), this leaves the higher-order dependencies abstract (meaning as RTD chooses). In [the log](https://readthedocs.org/api/v2/build/8954667.txt) I can see RTD has uninstalled its choice of sphinx 1.8.5, and installed PyPA's choice of 1.8.3, but most of its requirements are satisfied from the earlier action. This is not wholeheartedly either of the positions I identify.